### DISCUSSION TITLE
System cost with exitsing cappayment

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#109] [Q&A: Data]
Good afternoon! When I checked the modeling results via bokeh viewer, I was confused between the 'system cost annualized (billion $)' and 'system cost annualized including the existing (billion $)'. Is the difference between them  that one includes pre-2010 capital costs and the other does not? 

cappayment.csv seems to be the data source for calculating existing capacity payments, what is the unit in cappayment.csv? What is the calculation equation used for 'system cost annualized including the existing'? Thank you very much!

**Comment by wesleyjcole** [comment]
In reeds2.py (https://github.com/NREL/ReEDS_OpenAccess/blob/main/bokehpivot/reeds2.py) there is a remove_existing flag that handles whether or not existing capacity is included in the system cost calculation.  It notes a cost of billion $ (e.g., see line 142).
